xcdat/temporal.py

```diff
@@ -539,6 +548,11 @@ def departures(
         keep_weights : bool, optional
             If calculating averages using weights, keep the weights in the
             final dataset output, by default False.
+        reference_period : Optional[Tuple[str, str]], optional
+            The climatological reference period, which is a subset of the entire
+            time series and used for calculating departures. This parameter
+            accepts a tuple of strings in the format 'yyyy-mm-dd'. For example,
+            ``('1850-01-01', 1899-12-31')``.
```

Comment:
```suggestion
        reference_period : Optional[Tuple[str, str]], optional
            The climatological reference period, which is a subset of the entire
            time series and used for calculating departures. This parameter
            accepts a tuple of strings in the format 'yyyy-mm-dd'. For example,
            ``('1850-01-01', 1899-12-31')``. If no value is provided, the 
            climatological reference period will be the full period covered by 
            the dataset.
```

Simply adds: "If no value is provided, the climatological reference period will be the full period covered by the dataset."